index.bs

```diff
@@ -136,77 +136,156 @@ important to consider ways to mitigate the obvious impacts. For instance:
+
+When a feature exposes sensitive information to the web,
+its designers must take steps
+to mitigate the risk of exposing the information.
+
+<div class=example>
+
+The Credential Management API allows sites
+to request a user's credentials
+from a password manager. [[CREDENTIAL-MANAGEMENT-1]]
+If it exposed the user's credentials to JavaScript,
+and if the page using the API were vulnerable to [=XSS=] attacks,
+the user's credentials could be leaked to attackers.
+
+The Credential Management API
+mitigates this risk
+by not exposing the credentials to JavaScript.
+Instead, it exposes
+an opaque {{FormData}} object
+which cannot be read by JavaScript.
+The spec also recommends
+that sites configure Content Security Policy [[CSP]]
+with reasonable [=connect-src=] and [=form-action=] values
+to further mitigate the risk of exfiltration.
+
+</div>
+
+Many use cases
+which require location information
+can be adequately served
+with very coarse location data.
+For instance,
+a site which recommends restaurants
+could adequately serve its users
+with city-level location information
+instead of exposing the user's precise location.
+
+See also
+
+* [[DESIGN-PRINCIPLES#do-not-expose-use-of-assistive-tech]]
 
 <h3 class=question id="persistent-origin-specific-state">
   Does this specification introduce new state for an origin that persists
   across browsing sessions?
 </h3>
 
-Allowing an origin to persist data on a user’s device across browsing
-sessions introduces the risk that this state may be used to track a user
-without their knowledge or control, either in a first party or third party
-contexts.  New state persistence mechanisms should not be introduced without
-mitigations to prevent it from being used to track users across domains or
-without control over clearing this state.  And, are there specific caches
-that a user agent should specially consider?
-
-<p class=example>
-Service Worker [[SERVICE-WORKERS]] intercept all requests made by an
-origin, allowing sites to function perfectly even when offline. A
-maliciously-injected service worker, however, would be devastating (as
-documented in [[SERVICE-WORKERS#security-considerations]]).
-They mitigate the risks an [=active network attacker=] or [=XSS=]
-vulnerability present by requiring an encrypted and authenticated
-connection in order to register a service worker.
-</p>
-
-<p class=example>
-Platform-specific DRM implementations might expose origin-specific
-information in order to help identify users and determine whether they
-ought to be granted access to a specific piece of media. These kinds of
-identifiers should be carefully evaluated to determine how abuse can be
-mitigated; identifiers which a user cannot easily change are very
-valuable from a tracking perspective, and protecting the identifiers from
-an active network attacker is an important concern.
-</p>
+There are many existing mechanisms
+origins can use to
+store information about a user—cookies,
```

Comment:
should "user—cookies, rest of list" be "user: cookies, rest of list"?  I'm not familiar with dashes being used this way

Reply:
Tweaked in 04e2674.